### ISSUE TITLE
Cannot build from `main` due to FontSettings issue

### ISSUE THREAD

**Issue by mullr** [lapce/floem#205]
```
error[E0063]: missing field `load_substitutions` in initializer of `FontSettings`
  --> /home/mullr/.cargo/git/checkouts/vger-rs-7a55cf967af2d3a8/ed10537/src/glyphs.rs:53:24
   |
53 |         let settings = fontdue::FontSettings {
   |                        ^^^^^^^^^^^^^^^^^^^^^ missing `load_substitutions`
```

This appears to be due to a breaking change in the `fontdue` dependency in the newly released version 0.7.4. Filed upstream as https://github.com/mooman219/fontdue/issues/146.

**Comment by mooman219** [comment]
Resolved, 0.7.4 is yanked and 0.8.0 is re-published,

**Comment by mullr** [comment]
Looks like that fixed it. Thanks @mooman219!]
